### PR TITLE
Add canonical-global-nav as a dependency

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add canonical-global-nav as a dependency

## QA

- Run 'npm install -g bower'
- Run 'bower install canonical-global-nav'
- Run './run'
- Pop on over to http://0.0.0.0:8015 and make sure the global nav is visible
- Check in l, m and s

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "global-nav": "0.0.2",
```
Comment (anthonydillon) [inline package.json]: We should only be using the bower package. We don't need the npm one here.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks like you have left in a npm install of the global nav. Can you remove this please.

**Comment by anthonydillon** [comment]
@nottrobin also here. I thought `./run` supported bower? Does it just need to be updated?

**Comment by grahambancroft** [comment]
[Add global nav](https://trello.com/c/0eSelS4K/3236-add-global-nav)

File: package.json (hunk 1 above)
Comment (grahambancroft) [reply package.json]: Removed

**Comment by grahambancroft** [comment]
I’ll check with Will or Robin re the ./run question

**Comment by anthonydillon** [comment]
The final think to make travis happy is to add:
https://github.com/canonical-websites/www.ubuntu.com/pull/1729/commits/00ddc12ce9b502454a8c788e0191f355511bf59c and https://github.com/canonical-websites/www.ubuntu.com/pull/1729/commits/436db453b738b2e9aa0cd84bec16f151568e7d7c

**Comment by anthonydillon** [comment]
@grahambancroft looks like you have some lint issue here.

**Comment by anthonydillon** [comment]
QA issue:
The margin on the global-nav on large screen size (>1000px) is missing margin so touching the edge of the screen.
welcome - developer 2

**Comment by grahambancroft** [comment]
both issues fixed

**Review by anthonydillon — APPROVED** [review]

**Comment by anthonydillon** [comment]
Nice one.

**Pull request merged**